x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/selectors.ts

```diff
@@ -188,6 +200,17 @@ export const entriesExist: (state: Immutable<TrustedAppsListPageState>) => boole
   }
 );
 
+export const prevEntriesExist: (
+  state: Immutable<TrustedAppsListPageState>
+) => boolean = createSelector(entriesExistState, (doEntriesExists) => {
+  return (
```

Comment:
I think there is a helper method in async resource that will get prior state and does all of these types of checks. See the `getLastLoadedListResourceState()` above which is using it